packages/iframe-wallet-poc/src/iframe/user.ts

```diff
@@ -155,8 +154,7 @@ export class User
   public generateObserverNotification(
     notification: cf.legacy.node.Notification
   ): any {
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll don't we know the type of the proposed state transition?